sale_order_line_input/models/sale_order.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    pricelist_id = fields.Many2one(
+        related="order_id.pricelist_id",
+        readonly=True,
+    )
+    force_company_id = fields.Many2one(
+        comodel_name="res.company",
+        string="Forced company",
+        compute="_compute_force_company_id",
+        readonly=False,
+        help="Technical field to force company or get it "
+        "from env user if order don't exist.",
+    )
+
+    @api.depends("order_id")
+    def _compute_force_company_id(self):
+        """Related company is not computed already when we click create new line"""
+        for line in self:
+            line.force_company_id = (
+                line.order_id.company_id
+                # Is not necessary use browse here
+                or self.env.context.get("with_company")
```

Comment:
Don't change original line:
```suggestion
                or self.env.context.get("force_company")
```

Reply:
Change done.